test/extension-tests/single-root-UI/test/envs-kits.test.ts

```diff
@@ -41,8 +41,6 @@ suite('Environment Variables in Variants', () => {
             await fs.rename(variantFileBackup, variantFile);
         }
 
-        await vscode.workspace.getConfiguration('cmake', vscode.workspace.workspaceFolders![0].uri).update('useCMakePresets', 'auto');
```

Comment:
why were we changing this setting to 'auto'? Was it redundant from the get-go? because I can see we call `update('useCMakePresets', 'never')` or `update('useCMakePresets', 'always')` in the beginning of the tests anyways.

Reply:
It's best for tests to clean up after themselves so that the state going into each test is known. I'm removing this as a workaround for now because we can't currently await the work that happens after the setting is changed.